### PR TITLE
Selenium wrapper set up

### PR DESCRIPTION
**Pull request by jfer-siteimprove**
Resolves [II-8315](https://siteimprove-wgs.atlassian.net/browse/II-8315 )

[II-8315]: https://siteimprove-wgs.atlassian.net/browse/II-8315?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Jym77** [comment]
!pr dedupe

File: packages/alfa-selenium/src/selenium.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+export namespace Selenium {
+  export async function toPage(driver: WebDriver): Promise<Page> {
+    const deviceJSON = await driver.executeScript(device.Native.fromWindow);
+    const alfaDevice = Device.from(deviceJSON as Device.JSON);
+
+    const documentJSON = await driver.executeScript(dom.Native.fromNode);
+
+    return Page.of(
+      Request.empty(),
+      Response.empty(),
```
Comment (rcj-siteimprove) [inline packages/alfa-selenium/src/selenium.ts:25]: Can we get the request and response more similar to how the web extension does it?

https://github.com/Siteimprove/alfa-web-extension/blob/5d2aff7e2f7539d473d3a1da5a6e22bdae53f498/src/api/actions/serialize/serialize-page.ts#L30-L41

I think this is missing from the other wrappers as well, so maybe it they all need to be fixed together in a separate PR.

Comment (Jym77) [reply packages/alfa-selenium/src/selenium.ts:25]: Yes, we'll handle it with II-8316.

**Review by rcj-siteimprove — DISMISSED** [review]
Looks good 👏🏼

File: packages/alfa-puppeteer/test/puppeteer.spec.ts [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+  const page = await browser.newPage();
+
+  // Navigate to the page to scrape
+  await page.goto(url.pathToFileURL(path.join(fixture, "page.html")).href);
+
+  // Retrieve the viewport dimensions
+  const { width, height } = await page.evaluate(() => ({
+    width: window.document.documentElement.clientWidth,
+    height: window.document.documentElement.clientHeight,
+  }));
```
Comment (Jym77) [inline packages/alfa-puppeteer/test/puppeteer.spec.ts:30]: **Question:** Are these stable enough under puppeteer to be able to hardcode them in tests?

**Review by rcj-siteimprove — APPROVED** [review]
